### ISSUE TITLE
Images, alt text, and figures should be supported in EPUB

### ISSUE THREAD

**Issue by GeorgeKerscher** [trypsynth/paperback#374]
In the test EPUB from epubtest.org, there is the title:
[Non-visual fundamental test book](https://github.com/daisy/epub-accessibility-tests/releases/download/non-visual-reading-2.0.1/Fundamental-Accessibility-Tests-Non-Visual-Reading-v2.0.1.epub)
In there is the test reading-310 Image alternate text reading. Paperback version   0.8.2  on Windows 11 fails this test.

I asked my Meta glasses if it saw a picture, and it reported that no picture was visible. I also use the NVDA g to go to a graphic, and it reported none.

I think Paperback should suport image presentation. The alt text should be read and if the image is in a figure with a figure caption, then that should also be available. While I can understand not showing images to blind people, the alt text and captions should be shown. Today, many people will use AI available to them to describe images, and having the image presented would enable this functionality.
